lib/journaled/relation_change_protection.rb

```diff
@@ -1,7 +1,15 @@
 module Journaled::RelationChangeProtection
-  def update_all(updates, force: false)
+  def update_all(updates, force: false) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/PerceivedComplexity
     unless force || !@klass.respond_to?(:journaled_attribute_names) || @klass.journaled_attribute_names.empty?
-      conflicting_journaled_attribute_names = @klass.journaled_attribute_names & updates.keys.map(&:to_sym)
+      conflicting_journaled_attribute_names = if updates.is_a?(Hash)
+                                                @klass.journaled_attribute_names & updates.keys.map(&:to_sym)
+                                              elsif updates.is_a?(String)
+                                                @klass.journaled_attribute_names.select do |a|
+                                                  updates.match?(/(^|\A|[ \.\"])#{a}[\z \"$]/)
```

Comment:
could probably do something like this as an alt: `/[^aA-zZ0-9']#{a}[^aA-zZ0-9']/`

Reply:
does anyone know how to put a `^` in a character class? i did this | thing because i couldn't figure it out `(^|\A|[ \.\"])`